src/atom_finder/classifier/operator-precedence.clj

```diff
@@ -105,12 +107,18 @@
 
 (defn group-pairs-in-node
   "Returns all operator group pairs in a node"
-  [node child-groups]
-  (let [node-group (operator-group node)]
+  [node child-nodes]
+  (let [node-group (operator-group node) child-groups (map operator-group child-nodes)]
     (cond
            (instance? IASTUnaryExpression node) 
            [[node-group (safe-nth child-groups 0)]]
 
+           ;;SPECIAL CASE 2: Two bitwise_bins that are not the same, note that the group pair name is not [:bitwise_bin and bitwise_bin]
+           (and (and (= :bitwise_bin node-group)
+                     (= :bitwise_bin (safe-nth child-groups 1)))
+                (not (= (.getOperator node) (.getOperator (safe-nth child-nodes 1)))))
+           [[:bitwise--bin :bitwise--bin]]
```

Comment:
Ok, now I understand, but its still confusing. What about renaming these to like
`[[:non-matching-bitwise :non-matching-bitwise]]`
or, since it's impossible to ever have only one appear alone, maybe just
`[[:non-matching-bitwise]]`
or something?